search/intermediate_result.cpp

```diff
@@ -279,18 +279,23 @@ void FillDetails(FeatureType & ft, Result::Details & details)
       }
     }
   }
-
-  details.m_isHotel = ftypes::IsHotelChecker::Instance()(ft);
+  
+  auto typesHolder = feature::TypesHolder(ft);
```

Comment:
nit: feature::TypesHolder const typesHolder(ft);

Reply:
Done

Reply:
No, this is a complete expression, no need assign:
```
feature::TypesHolder const typesHolder(ft);
```

Reply:
Fixed now